u3-views/views/video-home.hbs

```diff
@@ -0,0 +1,7 @@
+{{#>dbtube}}
+<div class="home-videos">
+  {{#each teases}}
+  {{>video-card tease=this}}
+  {{/each}}
```

Comment:
นี่เป็นอีกมุมมองหนึ่งที่ฉันจะใช้ `videos`/`video` หรือ `cards`/`card` (ดังที่กล่าวไว้ก่อนหน้านี้)
This is the other place where I would use either `videos`/`video` or `cards`/`card` (as mentioned earlier).

Reply:
FIXED